packages/transmute-cli/src/commands/debug/debug

```diff
@@ -0,0 +1,5 @@
+echo 'DEBUG: CLI Details'
+echo $(whoami)
+echo $PWD
+
```

Comment:
Can we make a note here about the code-splitting that there will be other debug commands in the future?

Reply:
yes